mep/pages/models.py

```diff
@@ -348,6 +354,11 @@ class Person(models.Model):
     def __str__(self):
         return self.name
 
+    @property
+    def name(self):
```

Comment:
This is a simple enough property, but please add a unit test.

We decided to make both name fields required, right? So we don't need to handle any optional missing data here.

Reply:
Probably makes sense to add a `lastname_first` property on the model instead of constructing it in the template. We should probably use lastname first in the html metadata also.